### PR TITLE
Dop 1554 sending using dopplermessageservice

### PR DESCRIPTION
**Pull request by leoslopez**
- Se agrega el comportamiento en DopplerMessageService para que envie notificaciones utilizando el cliente WebPush:
chrome_5NbpkTppZx
- Ademas de las propiedades ya utilizadas con firebase (title, body, image, y clickLink), se agregó el `icon` (**TODO**: analizar comportamiento en distintos dispositivos):
image

**TODO:** 
- los `WebPushSettings` deben guardarse con los secretos de swarm (será considerado en este PR)
- agregar tests, prioritariamente, a MessageController y DopplerMessageService. Tambien podrian agregarse a las clases asociadas a WebPushClient.
- probar performance, y analizar optimizaciones de envio (https://makingsense.atlassian.net/browse/DOP-1555)

### CODE REVIEW

File: Doppler.Push.Api/Services/DopplerMessageService.cs [hunk 1]
```diff
@@ -1,19 +1,96 @@
...
+                        Subscription = subscription,
+                    });
+                    allFailureCount += 1;
+                }
+                catch (Exception ex)
+                {
+                    allResponses.Add(new ResponseItem()
+                    {
+                        IsSuccess = false,
+                        Exception = new ExceptionItem()
```
Comment (mmosquera) [inline Doppler.Push.Api/Services/DopplerMessageService.cs:77]: if you dont log the errror maybe is better throw the entire excption

**Review by mmosquera — APPROVED** [review]
minor comments

File: Doppler.Push.Api/Services/DopplerMessageService.cs (hunk 1 above)
Comment (leoslopez) [reply Doppler.Push.Api/Services/DopplerMessageService.cs:77]: > if you dont log the errror maybe is better throw the entire excption

But if it is broken due particular situation related to the current subscription being processed? Would not be better to inform the error related to that in the responses list?

File: Doppler.Push.Api/Services/WebPushClient.cs [hunk 2]
```diff
@@ -140,8 +139,7 @@ public void SetVapidDetails(string subject, string publicKey, string privateKey)
                 {
                     if (!validOptionsKeys.Contains(key))
                     {
-                        throw new ArgumentException(key + " is an invalid options. The valid options are" +
-                                                    string.Join(",", validOptionsKeys));
+                        throw new ArgumentException(key + " is an invalid options. The valid options are" + string.Join(",", validOptionsKeys));
```
[the remark this answers is not in the document]
Comment (leoslopez) [reply Doppler.Push.Api/Services/WebPushClient.cs]: > you could use interpolated string as above

Done.

**Pull request merged**